operator/config/default/manager_patch.yaml

```diff
@@ -13,4 +13,4 @@ spec:
         args:
         - "--metrics-addr=127.0.0.1:8080"
         - "--enable-leader-election"
-        - "--tailing-sidecar-image=sumologic/tailing-sidecar:latest"
+        - "--tailing-sidecar-image=localhost:32000/sumologic/tailing-sidecar:latest"
```

Comment:
Is it intended?

Reply:
It should not have be added to this commit, thank for catching this